projects.md

```diff
@@ -1,3 +1,27 @@
+---
+layout: contribute
+title: Mentoring Projects
+---
+
+## Mentoring Projects
+
+This page details potential projects and their mentors, for participants of
+internship programs such as [Google Summer of Code](https://www.google-melange.com/),
+[Outreachy](https://www.gnome.org/outreachy/), and so forth. Each program will
+have it's own requirements and application process - this page simply lists the
+currently available projects.
+
+## I'm interested - how can I get familiar with Foreman?
+
+* Grab a cup of your favorite beverage
+* Skim through the [manual](/documentation.html)
+* Install Foreman on a virtual machine or your own computer
```

Comment:
https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/templates/web.conf.erb#L13

It might be possible to fix that now with jekyll-redirect-from.